pkgs/applications/video/mpv/default.nix

```diff
@@ -99,36 +100,33 @@ let
     else stdenv;
 in stdenv'.mkDerivation (finalAttrs: {
   pname = "mpv";
-  version = "0.36.0";
+  version = "0.37.0";
 
-  outputs = [ "out" "dev" "man" ];
+  outputs = [ "out" "doc" "man" ];
```

Comment:
Why remove the dev output?

Reply:
How?

Reply:
I did nix build --keep-failed and noticed that they code the dev output into the shared objects.